packages/notifi-react-example/src/EthosWalletProvider.tsx

```diff
@@ -0,0 +1,16 @@
+import { EthosConnectProvider } from 'ethos-connect';
+import { PropsWithChildren } from 'react';
+
+export const EthosWalletProvider: React.FC<PropsWithChildren> = ({
+  children,
+}) => {
+  return (
+    <EthosConnectProvider
+      ethosConfiguration={{
+        hideEmailSignIn: true, // defaults to false
```

Comment:
oo interesting. noticed that they had a sign up with email.

Reply:
Yea, not try it out yet. But it's interesting.